__docs__/wonder-blocks-labeled-field/react-hook-form.stories.tsx

```diff
@@ -0,0 +1,515 @@
+ *   - Text is < 3 or > 5 characters
+ * - SingleSelect, MultiSelect, RadioGroup, CheckboxGroup:
+ *   - No selected value (because it is required)
+ *   - Include "Banana" in the selection
+ */
+export const ExampleController = (args: StoryArgs) => {
+    const {
+        handleSubmit,
+        formState: {errors},
+        control,
+    } = useForm<Inputs>({
+        mode: args.mode,
+        defaultValues,
+        disabled: args.disableFields,
+    });
+
+    const onSubmit: SubmitHandler<Inputs> = (data) => {
+        console.log("successful submit", data);
+    };
+
+    const onInvalidSubmit: SubmitErrorHandler<Inputs> = (data) => {
+        console.log("invalid submit", data);
+    };
+
+    return (
+        <form onSubmit={handleSubmit(onSubmit, onInvalidSubmit)}>
+            <View style={{gap: spacing.medium_16}}>
+                <Controller
+                    name="exampleTextField"
+                    control={control}
+                    rules={textRules}
+                    render={({field}) => {
+                        return (
+                            <LabeledField
+                                label="TextField"
+                                description="Helpful description text."
+                                errorMessage={errors.exampleTextField?.message}
+                                field={<TextField {...field} />}
+                            />
+                        );
+                    }}
+                />
+                <Controller
+                    name="exampleTextArea"
+                    control={control}
+                    rules={textRules}
+                    render={({field}) => {
+                        return (
+                            <LabeledField
+                                label="TextArea"
+                                description="Helpful description text."
+                                errorMessage={errors.exampleTextArea?.message}
+                                field={<TextArea {...field} />}
+                            />
+                        );
+                    }}
+                />
+                <Controller
+                    name="exampleSingleSelect"
+                    control={control}
+                    rules={selectedValueRules}
+                    render={({field}) => {
+                        return (
+                            <LabeledField
+                                label="SingleSelect"
+                                description="Helpful description text."
+                                errorMessage={
+                                    errors.exampleSingleSelect?.message
+                                }
+                                field={
+                                    <SingleSelect
+                                        placeholder="Choose a fruit"
+                                        {...field} // NOTE: field includes onBlur, which SingleSelect doesn't support currently
```

Comment:
Since SingleSelect/MultiSelect don't currently support onBlur, on blur validation behaviour doesn't work as expected. This can be addressed by supporting on blur props for all field components